src/EnergyPlus/IndoorGreen.cc

```diff
@@ -0,0 +1,778 @@
+                ScheduleManager::GetScheduleIndex(state, state.dataIPShortCut->cAlphaArgs(3));
+            if (lw->indoorgreen(IndoorGreenNum).SchedPtr == 0) {
+                if (state.dataIPShortCut->lAlphaFieldBlanks(3)) {
+                    ShowSevereError(state,
+                                    format("{} =\"{}\", {} is required.",
+                                           RoutineName,
+                                           state.dataIPShortCut->cAlphaArgs(1),
+                                           state.dataIPShortCut->cAlphaFieldNames(3)));
+                } else {
+                    ShowSevereError(state,
+                                    format("{} =\"{}\", invalid {} entered={}",
+                                           RoutineName,
+                                           state.dataIPShortCut->cAlphaArgs(1),
+                                           state.dataIPShortCut->cAlphaFieldNames(3),
+                                           state.dataIPShortCut->cAlphaArgs(3)));
+                }
+                ErrorsFound = true;
+            } else { // check min/max on schedule
+                SchMin = ScheduleManager::GetScheduleMinValue(state, lw->indoorgreen(IndoorGreenNum).SchedPtr);
+                SchMax = ScheduleManager::GetScheduleMaxValue(state, lw->indoorgreen(IndoorGreenNum).SchedPtr);
+                if (SchMin < 0.0 || SchMax < 0.0) {
+                    if (SchMin < 0.0) {
+                        ShowSevereError(state,
+                                        format("{}=\"{}\", {}, minimum is < 0.0",
+                                               RoutineName,
+                                               state.dataIPShortCut->cAlphaArgs(1),
+                                               state.dataIPShortCut->cAlphaFieldNames(3)));
+                        ShowContinueError(
+                            state,
+                            format("Schedule=\"{}\". Minimum is [{:.1R}]. Values must be >= 0.0.", state.dataIPShortCut->cAlphaArgs(3), SchMin));
+                        ErrorsFound = true;
+                    }
+                    if (SchMax < 0.0) {
+                        ShowSevereError(state,
+                                        format("{}=\"{}\", {}, maximum is < 0.0",
+                                               RoutineName,
+                                               state.dataIPShortCut->cAlphaArgs(1),
+                                               state.dataIPShortCut->cAlphaFieldNames(3)));
+                        ShowContinueError(
+                            state,
+                            format("Schedule=\"{}\". Maximum is [{:.1R}]. Values must be >= 0.0.", state.dataIPShortCut->cAlphaArgs(3), SchMin));
+                        ErrorsFound = true;
+                    }
+                }
+            }
+            lw->indoorgreen(IndoorGreenNum).ETCalculationMethod = 1; // default
```

Comment:
Make these into an `enum` and use `getEnumValue`.

Reply:
Thank you! Changed to Enum for ETCalculationMethod.